skimage/draw/draw.py

```diff
@@ -41,35 +41,35 @@ def _ellipse_in_shape(shape, center, radii, rotation=0.):
     return np.nonzero(distances < 1)
 
 
-def ellipse(r, c, r_radius, c_radius, shape=None, rotation=0.):
-    """Generate coordinates of pixels within ellipse.
+def ellipse(r, c, r_radius, c_radius, orientation=0, shape=None, rotation=None):
```

Comment:
This is an API change... Are we not going to have a full deprecation path here? I don't see how the code handles the fact that you can now get orientation be shape and shape be orientation...?

Reply:
Come to think of it, could we go all the way with this: make ellipse be nD compatible with ellipsoid, and have the input be `coord, radii, orientation, shape`?

Reply:
I was checking `ellipsoid`, and it returns an array instead of the coordinates. Do we have any idea why?

Reply:
(I'd go for `center` instead of `coords`, but I'm +1 on joining the variables)

Reply:
>  I don't see how the code handles the fact that you can now get orientation be shape and shape be orientation...?

@jni yep, I'll need some help on this... however, first we need to discuss the API of drawing-related stuff and then we can tackle that, ok?

Reply:
Yep. I agree with you on `center`, good call. I'd support changing ellipsoid to return coordinates.

Reply:
@jni what if we started that new functions with the desirable API changes within `future`?